bamboo/docker-compose.yml

```diff
@@ -15,7 +15,7 @@ services:
       - FTP_PASSWORD=testpass
       - FTP_PASV_ADDRESS=127.0.0.1
   http:
-    image: maven.earthdata.nasa.gov/httpd:alpine
+    image: httpd:alpine
```

Comment:
Updated this per convo with CICD team, updating the maven image is no longer required as Bamboo is now configured licensed Dockerhub account.